tests/baselines/reference/generatorTypeCheck48.errors.txt

```diff
@@ -1,9 +0,0 @@
-tests/cases/conformance/es6/yieldExpressions/generatorTypeCheck48.ts(1,9): error TS7025: Generator implicitly has type 'IterableIterator<any>' because it does not yield any values. Consider supplying a return type.
-
-
-==== tests/cases/conformance/es6/yieldExpressions/generatorTypeCheck48.ts (1 errors) ====
-    function* g() {
-            ~
-!!! error TS7025: Generator implicitly has type 'IterableIterator<any>' because it does not yield any values. Consider supplying a return type.

```

Comment:
this was the correct error. without noImplicitAny, it should be an implicit any and not `undefined`.